docs/dotnet/mvvm/Introduction.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Introduction to the MVVM Toolkit
+author: Sergio0694
+description: An overview of how to get started with the MVVM Toolkit and to the APIs it contains
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, get started, visual studio, MVVM, net core, net standard
+dev_langs:
+  - csharp
+  - vb 
+---
+
+# Introduction to the MVVM Toolkit
+
+The `Microsoft.Toolkit.Mvvm` package (aka MVVM Toolkit) is a modern, fast, and modular MVVM library. It is part of the Windows Community Toolkit and is built around the following principles:
+
+- **Platform and Runtime Independent** - **.NET Standard 2.0** and **.NET 5** 🚀 (UI Framework Agnostic)
+- **Simple to pick-up and use** - No strict requirements on Application structure or coding-paradigms (outside of 'MVVM'ness), i.e., flexible usage.
+- **À la carte** - Freedom to choose which components to use.
+- **Reference Implementation** - Lean and performant, providing implementations for interfaces that are included in the Base Class Library, but lack concrete types to use them directly.
+
+This package targets **.NET Standard** so it can be used on any app platform: UWP, WinForms, WPF, Xamarin, Uno, and more; and on any runtime: .NET Native, .NET Core, .NET Framework, or Mono. It runs on all of them. The API surface is identical in all cases, making it perfect for building shared libraries.
+
+Additionally, the MVVM Toolkit also has a **.NET 5** target, which is used to enable more internal optimizations when running in on the .NET 5 runtime. The public API surface is identical in both cases, so NuGet will always resolve the best possible version of the package without consumers having to worry about which APIs will be available on their platform.
+
+To install the package from within Visual Studio:
+
+1. In Solution Explorer, right-click on the project and select **Manage NuGet Packages**. Search for **Microsoft.Toolkit.Mvvm** and install it.
+
+    ![NuGet Packages](../images/get-started/manage-nuget-packages.png "Manage NuGet Packages Image")
```

Comment:
Maybe we should create an issue around having a centralized managing NuGet package page we can link to across all toolkit getting started docs?

Reply:
Yup, that'd be great. No point in duplicating this 🙂

Reply:
Opened #52